pkg/helmexec/helmexec.go

```diff
@@ -1,5 +1,7 @@
 package helmexec
 
+import "helm.sh/helm/v3/pkg/chart"
```

Comment:
I think this is our first attempt to use Go helm pkg directly rather than calling out to the helm command.
I don't know if it works out in the long term but it would be worth trying.

Reply:
@yxxhero If this is okay, we might be able to replace every invocation to helm command with the respective call to `helm.sh/helm/v3/pkg/whatever` structs and functions. That would free our users from being required to make Helmfile and Helm versions in-sync. WDYT?

Reply:
This is equivalent to not relying on the help binary, which should require many changes and can be optimized in the future. @mumoshu